mongo/mongo.go

```diff
@@ -302,7 +302,7 @@ func (m *Mongo) roundTrip(conn driver.Connection, req []byte, unacknowledged boo
 		return nil, nil
 	}
 
-	if res, err = conn.ReadWireMessage(m.roundTripCtx, req[:0]); err != nil {
+	if res, err = conn.ReadWireMessage(m.roundTripCtx); err != nil {
```

Comment:
Latest driver change will now self-create the byte array when reading the wire message: https://github.com/mongodb/mongo-go-driver/blob/v1/x/mongo/driver/topology/connection.go#L471